packages/x-charts/src/ChartsTooltip/useItemTooltip.tsx

```diff
@@ -47,16 +42,25 @@ export function useInternalItemTooltip<T extends ChartSeriesType>():
     | ChartSeriesDefaultized<T>
     | undefined;
 
+  const xAxisId: AxisId | undefined =
+    itemSeries && isCartesianSeries(itemSeries) ? (itemSeries.xAxisId ?? xAxisIds[0]) : undefined;
+  const yAxisId: AxisId | undefined =
+    itemSeries && isCartesianSeries(itemSeries) ? (itemSeries.yAxisId ?? yAxisIds[0]) : undefined;
+  const zAxisId: AxisId | undefined =
+    itemSeries && 'zAxisId' in itemSeries ? (itemSeries.zAxisId ?? zAxisIds[0]) : zAxisIds[0];
+
+  const rotationAxisId: AxisId | undefined = rotationAxisIds[0];
```

Comment:
Why do we always use the first rotation axis? Can the chart have more than one?

Reply:
The only polar chart we currently have is the radar which does not have axis ids. The radar internally define its axes and rely on their order (which is not an issue since the user should not touch to them